### PR TITLE
Add AmazonChimeSDKMediaNoVideoCodecs to SPM

### PR DESCRIPTION
**Pull request by bjason**
*Issue #, if available:*

*Description of changes:*

Onboard AmazonChimeSDKMediaNoVideoCodecs to SPM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -12,6 +12,7 @@ let hostingUrl = "https://amazon-chime-sdk-ios.s3.amazonaws.com"
 // Checksums for SDKs to verify that the hosted archive file matches the archive declared in the manifest file.
 let SDKChecksum = "bc4193855ec5b96732846b27954573ee4b9ec497f930e9a678125ee5b3a8f49a"
 let SDKMediaChecksum = "03751799d43c5325524f9056057530f25da4ccdc6d712acb24f556471f0e939a"
+let SDKMediaNoVideoCodecsChecksum = "03751799d43c5325524f9056057530f25da4ccdc6d712acb24f556471f0e939a"
```
Comment (bjason) [inline Package.swift:15]: This value is randomly set for now and will be updated once we run the sdk_spm_version_update script

Comment (linsang21) [reply Package.swift:15]: can we run script first?

Comment (georgezy-amzn) [reply Package.swift:15]: I'm thinking of if we should set it to real value.

File: Package.swift [hunk 2]
```diff
@@ -32,6 +33,11 @@ let package = Package(
             url: "\(hostingUrl)/media-without-bitcode/\(SDKMediaVersion)/spm/AmazonChimeSDKMedia-\(SDKMediaVersion).zip",
             checksum: SDKMediaChecksum
         ),
+        .binaryTarget(
+            name: "AmazonChimeSDKMediaNoVideoCodecs",
+            url: "\(hostingUrl)/media-no-video-codecs-without-bitcode/\(SDKMediaVersion)/spm/AmazonChimeSDKMediaNoVideoCodecs-\(SDKMediaVersion).zip",
+            checksum: SDKMediaChecksum
```
Comment (linsang21) [inline Package.swift:39]: should it be `SDKMediaNoVideoCodecsChecksum`?

**Comment by bjason** [comment]
Closing this PR as the same reason listed here: https://github.com/aws/amazon-chime-sdk-ios-spm/pull/7

**Pull request closed without merging**